.github/workflows/scripts/add-codeowners-to-pr.sh

```diff
@@ -104,6 +104,10 @@ main () {
         done
     done
 
+    if [[ $LABELS =~ "receiver/sqlserver" ]]; then
+      LABELS+=",Run Windows"
```

Comment:
- You can add the same label multiple times in the `gh` CLI, it will only get added in GitHub once (in other words, adding multiple times is the same operation as once, it's an idempotent operation.)
- Tested on another repo, adding labels with spaces works without issue.
- Tested on another repo, there must not be a space after the delimiting comma.